Azure-Samples/azure-cosmos-db-graph-npm-bom-sample
# Added a Dotnet Core Database Loader Console App, updated the Portal screen shots and related README

## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
See the instructions in the README
```

* Test the code

```
See the instructions in the README re: running the web app locally
```

## What to Check
Verify that the following are valid
* ...

## Other Information
